.gitignore

```diff
@@ -1,2 +1,3 @@
 lib
 umd
+node_modules
```

Comment:
Please remove this and put it in your `~/.gitignore` instead.

Reply:
testing